Library/Formula/consul.rb

```diff
@@ -13,6 +13,8 @@ class Consul < Formula
     sha256 "67052157266d536598ba7eb5af31a10fd3e609ca812be6d50869d0fe80a583f9" => :mountain_lion
   end
 
+  option "with-web-ui", "Installs the consul web ui (to /usr/local/share/consul directory)"
```

Comment:
My gut feeling is that it's easy enough to work out where it has been installed without needing to include that in the option, but I'll let some of the other maintainers comment on this if they'd like to.